### ISSUE TITLE
What is the pattern for SFTP?

### ISSUE THREAD

**Issue by omexlu** [emuell/restic-browser#128]
Hi,

I downloaded Restic Browser, works fine when it's local.

Now I want to access sftp, at the bottom it says fetching snapshots but nothing comes in, just loads.

sftp://USER@USER.your-storagebox.de//REPO

Am I doing something wrong here?

**Comment by emuell** [comment]
How exactly do you pass the SFTP password to restic? Do you enter it interactively in the shell with restic? This won't work in Restic Browser, because there is no shell. You could either use ssh or rclone to avoid entering a password interactively.

Also note that the sftp path in Restic browser should be exactly the same as with the restic CLI, but without the `sftp:` prefix.

```
USER@USER.your-storagebox.de:REPO
````

**Comment by omexlu** [comment]
Hi @emuell 

I open Restic-Browser select " SFTP ", enter " USER@USER.your-storagebox.de:REPO ", enter " Repo-Password ", click on " Okay ", after that it shows me on the left-bottom corner " Fetching snapshots" but nothing happends, it loads but nothing coming in, even after 10-15 min.

If i use the restic in command-line i need first to enter " SFTP-password " then i get a prompt to enter " Repo-password ".

Maybe i need to pass the SFTP-Password to in Restic-Browser but how?

Thanks in advance.

**Comment by emuell** [comment]
> Maybe i need to pass the SFTP-Password to in Restic-Browser but how?

There is no shell in Restic Browser to do that. But you can use [rclone](https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#other-services-via-rclone) or set a [ssh key for your storage box](https://docs.hetzner.com/storage/storage-box/backup-space-ssh-keys/) to avoid to entering a password for sftp.

I'll check if there's a way to generate a better error in restic browser to make that more clear.

**Comment by omexlu** [comment]
Thank you, after creating and uploading the ssh-key to the storagebox, restic-browser will automatically search for the key or anything needs to be configured in connection field or somewhere else in restic-browser?

**Comment by emuell** [comment]
Yes, it *should* work just fine then. That's how I use it on Windows. It may be problematic on macOS.

If it does not, use rclone. That definitely will work.
